dalston/block-extends/extend-cover-block.css

```diff
@@ -2,10 +2,10 @@
  * Editor
  */
 
-#editor .wp-block-cover-image.is-style-hover-reveal.has-background-dim.has-background-dim .wp-block-cover__gradient-background,
```

Comment:
Just remove the not needed duplicate selector `.has-background-dim.has-background-dim`.